pythran/tests/test_spec_parser.py

```diff
@@ -50,3 +50,39 @@ def test_invalid_specs2(self):
         code = '#pythran export bar(int)\ndef foo(): pass'
         with self.assertRaises(pythran.syntax.PythranSyntaxError):
             pythran.compile_pythrancode("dumber", code)
+
+    def test_multiline_spec0(self):
+        code = '''
+#pythran export foo(
+              # )
+def foo(): return
+            '''
+        pythran.spec_parser(code)
+
+    def test_multiline_spec1(self):
+        code = '''
+#pythran export foo(int
+  #, int
+              # )
+def foo(i,j): return
+            '''
+        pythran.spec_parser(code)
+
+    def test_multiline_spec2(self):
+        code = '''
+#      pythran export foo(int,
+
+# float
+  #, int
+              # )
+def foo(i,j,k): return
+            '''
+        pythran.spec_parser(code)
+
+    def test_crappy_spec0(self):
+        code = '''
+#      pythran export foo(int) this is a pythran test
+def foo(i,j,k): return
+            '''
+        pythran.spec_parser(code)
+
```

Comment:
On Sat, Dec 05, 2015 at 10:41:09AM -0800, pbrunet wrote:
> Do we need a test with a comment on a next line? (I guess it is the same as crappy export)
Better safe than sorry, so let's add it :-)